app/models/item.rb

```diff
@@ -22,10 +22,7 @@ def self.most_revenue(quantity)
 
   def self.top_items_by_number_sold(quantity)
     unscoped.
-      select(
-        "items.*, "\
-        "SUM(invoice_items.quantity) AS most_items"
-      ).
```

Comment:
Made this a single line as it was under the 80 char limit and was more readable.